From 3c1a3571acaae24c1be6eeed66d2687001c14f2b Mon Sep 17 00:00:00 2001
From: Amarnath Valluri <amarnath.valluri@intel.com>
Date: Thu, 7 Jan 2016 16:19:34 +0200
Subject: [PATCH swupd-client 2/5] Fix build failure on Yocto

On install phase certificate files are being installed twice as included in
_DATA twice. We can use EXTRA_DIST than dist_.

Upstream-Status: Submitted [https://github.com/clearlinux/swupd-client/pull/9]

Signed-off-by: Amarnath Valluri <amarnath.valluri@intel.com>
---
 Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 1e65d3d..4d15c55 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -111,11 +111,11 @@ SWUPD_CERTS = certs/157753a5.0 \
 	certs/d6325660.0 \
 	certs/d6325660.1
 swupdcerts_DATA = $(SWUPD_CERTS)
-dist_swupdcerts_DATA = $(SWUPD_CERTS)
 
 EXTRA_DIST += \
 	data/check-update.service \
-	data/check-update.timer
+	data/check-update.timer \
+	$(SWUPD_CERTS)
 
 DISTCHECK_CONFIGURE_FLAGS = \
         --with-systemdsystemunitdir=$$dc_install_base/$(systemdunitdir)
-- 
2.5.0

